packages/api/tests/lambdas/test-sf-starter.js

```diff
@@ -299,72 +292,6 @@ test('handleThrottledEvent starts MAX - N executions for messages with priority'
   t.is(messages.length, numOfMessages - result);
 });
 
-test('handleThrottledEvent respects maximum executions for multiple priority levels', async (t) => {
```

Comment:
Great catch. It was removed because the test was actually verifying a behavior that was deprecated and no longer supported - but because of the test setup it still worked. 

When I first implemented execution throttling, there was this idea of messages having "priority levels" each with an execution limit. Thus, messages sent to the same queue could be throttled differently based on their priority level. But I think we quickly decided that behavior was YAGNI and the execution limit should be associated with a queue. 

And despite support for "priority levels" being removed, this test was still passing because it was creating workflow message fixtures that referred to different queues (in cumulus_meta.queueName) even though the messages were being sent to the same queue, so the test was still passing based on normal throttling by queue. But not only does this test setup not make sense, it can't happen in reality. Since the test was for a deprecated feature and confusing all around, I decided to remove it

Reply:
Cool - I inferred some of that looking at the test, just wanted to make sure it was intended.